TESTING_MACOS_AND_IOS.md

```diff
@@ -0,0 +1,75 @@
+# Test MacOS and iOS app on windows/linux
+
+If you need help, have any questions or suggestions, please ask in the [`#expensify-open-source` Slack channel](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions).
+
+## Problem
+MacOS is needed for testing New Expensify on iOS and MacOS. They are the most important platforms for Expensify. But, many issues aren’t being tested on MacOS and iOS by the contributors because they have Windows or Linux machines, and sometimes, proposals aren’t made with keeping these platforms in mind. 
+
+## Solution
+Majority of the contributors need MacOS for short bursts of time for the purpose of testing. Cloud services that offer MacOS instances like AWS, MacInCloud, MacStadium etc have a very high minimum cost. [Scaleway’s M1 Mac minis](https://www.scaleway.com/en/hello-m1/) provide dedicated instances at just $2.7 per day.
+
+
+| **Cloud provider**|**Minimum cost** |
+| ----------- | ----------- |
+| **Scaleway** | **$2.7 per day 🥇** |
+| MacInCloud |$19 for 5 days, or **$35 per month** |
+| MacStadium | $132 per month |
+| AWS | $26 per day |
+
```

Comment:
I think we should remove these lines because this document is now intended to serve as a quick set up guide. The reader already knows that they need to test on iOS and MacOS. Perhaps we could leave the solution portion for more context about why we chose Scaleway, but I think that is probably unnecessary.

Reply:
Yeah, makes sense. I think we should keep the solution part tho.
Because I'm sure contributors will explore other cloud providers and this will just save them some time.